tests/phpunit/test-class-wc-stripe-notes.php

```diff
@@ -29,7 +29,6 @@ public function setUp() {
 			return;
 		}
 
-		update_option( '_wcstripe_feature_upe_settings', 'yes' );
```

Comment:
Do the changes in this file (and other similar test files) affect the tests in any way?

Reply:
Not on this file neither in `tests/phpunit/test-wc-stripe-payment-gateway.php` Do you want me to revert the changes for these files?

Only the changes in `tests/phpunit/test-wc-stripe-upe-compatibility-controller.php` are needed for the tests to pass.

Reply:
> Do you want me to revert the changes for these files?

No, I was just wondering if changing the options in the tests would have an affect on the behavior during the tests. It's a little bit weird that we can just remove the option without changing the tests and they still pass 😛 

I think keeping the changes as they are is fine ☺️